tests/test_components/test_geometry.py

```diff
@@ -487,3 +488,32 @@ def test_gds_cell():
     td.PolySlab.from_gds(gds_cell=gds_cell, axis=2, slab_bounds=(-1, 1), gds_layer=0)
     with pytest.raises(Tidy3dKeyError):
         td.PolySlab.from_gds(gds_cell=gds_cell, axis=2, slab_bounds=(-1, 1), gds_layer=1)
+
+
+def test_custom_surface_geometry():
```

Comment:
Let's add the other methods (._normals, etc) to the tests. Also, we should add a CustomSurfaceGeometry to the [SIM_FULL](https://github.com/flexcompute/tidy3d/blob/develop/tests/utils.py#L59) in tests/utils.py and run all of the tests as that will really test the file writing and other things that we need to make sure work as well.